src/objects/Rollup__mdt.object

```diff
@@ -131,6 +106,7 @@
     </fields>
     <fields>
         <fullName>Integer__c</fullName>
+        <description>Integer value to indicate how many years ago or days back to roll-up for</description>
```

Comment:
@njjc I think we're missing the corresponding field on this object to indicate the "Yearly Operation Type". The writeup indicate a pick-list with values for All Time/Yearly/N Days Back (https://salesforce.quip.com/DcFZApxrEY3I).

Reply:
Yep, let's add it.